packages/docusaurus-theme-classic/src/theme/Layout/styles.module.css

```diff
@@ -11,7 +11,7 @@ body {
 }
 
 .mainWrapper {
-  flex: 1 0 auto;
+  flex: 1 0 none;
```

Comment:
curious, do you have and ref doc mentioning this "none" value?

According to what I find it's only valid when using `flex: 'none'` but not as a 3rd position?

Reply:
Sorry, I miss that thing... 
trying again